app/backend/approaches/prompts/chat/query_few_shots.jinja

```diff
@@ -0,0 +1,18 @@
+[
+    {
```

Comment:
Interesting, I hadn't considered making the few shots a Jinja as well. Do you imagine passing template variables into it?

Reply:
This  makes more sense indeed. I converted it to .jinja initially to stay consistent with the file extensions in the prompts folder. I've now updated these to .json.